resources/js/components/view/DataImporter.vue

```diff
@@ -0,0 +1,668 @@
+                        </div>
+                        <div
+                            v-else
+                            class="alert alert-primary"
+                        >
+                            {{ t("main.importer.info.entity_type_has_no_attributes") }}
+                        </div>
+                    </div>
+                </div>
+                <importer-update-state
+                    v-if="state.validated"
+                    :conflict="state.validationData.conflict"
+                    :update="state.validationData.update"
+                    :create="state.validationData.create"
+                />
+                <footer class="d-flex align-items-center justify-content-between">
+                    <button
+                        class="btn btn-danger"
+                        @click="cancelImport"
+                    >
+                        {{ t(`global.cancel`) }}
+                    </button>
+
+                    <LoadingButton
+                        v-if="!state.validated"
+                        :loading="state.validating"
+                        class="btn btn-primary"
+                        :disabled="!canValidate || state.validating"
+                        @click="validate"
+                    >
+                        {{ t(`main.importer.validate`) }}
+                    </LoadingButton>
+                    <LoadingButton
+                        v-else
+                        :loading="state.uploading"
+                        class="btn btn-primary"
+                        :disabled="!canImport || state.uploading"
+                        @click="upload"
+                    >
+                        {{ t(`main.importer.import_btn`) }}
+                    </LoadingButton>
+                </footer>
+            </div>
+        </div>
+    </div>
+</template>
+
+
+<script>
+    import {
+        computed,
+        reactive,
+        watch,
+    } from 'vue';
+
+    import store from '%store';
+
+    import { stringSimilarity } from 'string-similarity-js';
+
+    import VueUploadComponent from 'vue-upload-component';
+    import CsvTable from '../CsvTable.vue';
+    import ImporterUpdateState from '../tools/importer/ImporterUpdateState.vue';
+    import EntityImporterSettings from '@/components/tools/importer/EntityImporterSettings.vue';
+    import EntityAttributeMapping from '../tools/importer/EntityAttributeMapping.vue';
+
+    import { useI18n } from 'vue-i18n';
+    import { useToast } from '@/plugins/toast.js';
+
+    import {
+        importEntityData,
+        validateEntityData,
+    } from '@/api.js';
+
+    import {
+        showImportError,
+    } from '@/helpers/modal.js';
+
+    import {
+        translateConcept,
+    } from '@/helpers/helpers.js';
+
+    import {
+        useOptionalLocalStorage
+    } from '../../composables/local-storage';
+    import { onMounted } from 'vue';
```

Comment:
`vue` already used on line 138-142

Reply:
Fixed in 65c9f6f2d62023b3a97b4035efd9253701cc072a